stdlib/builtins.pyi

```diff
@@ -1347,7 +1354,7 @@ else:
     @overload
     def sum(__iterable: Iterable[_T], __start: _S) -> _T | _S: ...
 
-def vars(__object: Any = ...) -> dict[str, Any]: ...
+def vars(__object: object = ...) -> dict[str, Any]: ...
```

Comment:
```python
vars(3)  # TypeError
```

Reply:
Dang, of course, it has to have a `__dict__` attribute for `vars()` to work.

Reply:
Shame this doesn't seem to work https://mypy-play.net/?mypy=latest&python=3.10&flags=show-error-codes%2Cstrict&gist=61cabb58a0f6ad34bf51f1b0e03afe11

Reply:
I think `__dict__` is too special, although ideally that would work.